docs/source/chapter1.rst

```diff
@@ -0,0 +1,136 @@
+|RAT banner|
+
+.. _chapter1:
+
+Chapter 1 - Introduction.
+.........................
+
+1. What is RAT.
```

Comment:
Please remove the title numbers and try [sectnum](https://docutils.sourceforge.io/docs/ref/rst/directives.html#automatic-section-numbering) for this instead

Reply:
Also you can remove the "Chapter"

Reply:
Done.